### PR TITLE
feat(cvss-validation): validating vulnerability.json CVSS field inputs

### PR DESCRIPTION
**Pull request by benharvie**
- Checks CVSS vectors for valid inputs

Example job: https://github.com/benharvie/huntr/runs/1887792002?check_suite_focus=true

### CODE REVIEW

File: .github/workflows/disclosure-submitted.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ on:
   pull_request_target:
     types: [opened, synchronize, reopened]
     branches:
-      - staging
+      - feat/cvss-validation # staging
```
Comment (JamieSlome) [inline .github/workflows/disclosure-submitted.yml:6]: Back to `staging` pls.

Comment (benharvie) [reply .github/workflows/disclosure-submitted.yml]: Sorted, ty 👻

File: .github/workflows/disclosure-submitted.yml [hunk 2]
```diff
@@ -303,6 +331,7 @@ jobs:
     needs:
       [
         contains-a-readme,
+        contains-a-valid-cvss,
```
Comment (JamieSlome) [inline .github/workflows/disclosure-submitted.yml:334]: We can still track a disclosure submitted without a valid CVSS - remove this line.

Comment (benharvie) [reply .github/workflows/disclosure-submitted.yml]: Removed 👍🏼

**Comment by JamieSlome** [comment]
@benharvie - can we prevent changes to the `generate-diff` file and `package-lock.json`?

File: tools/disclosure-submitted/generate-diff.js [hunk 3]
```diff
@@ -7,39 +7,45 @@ const octokit = new Octokit();
...
-
-    diff.push({
-        change: change,
-        path: file.filename
-    });
-})
+let diff = [];
 
-console.log(`The diff generated by this PR ${PR_NUMBER} is: \n`, diff)
+files.forEach((file) => {
```
Comment (benharvie) [inline tools/disclosure-submitted/generate-diff.js:33]: @JamieSlome - I'm unsure as to where these additions were made from, they weren't from myself. Let me know if you would like it included or not and I can have it modified, ty x

Comment (JamieSlome) [reply tools/disclosure-submitted/generate-diff.js:33]: @benharvie - I would say restore these to the current versions on staging - same for the `package-lock.json`.

Comment (benharvie) [reply tools/disclosure-submitted/generate-diff.js:33]: package-lock has been done, the changes on generate-diff are now only linting changes when compared to `staging`, cheers 👻

**Review by JamieSlome — APPROVED** [review]
LGTM! 🍰

**Pull request merged**